tests/test_xyz/test_points_from_wells.py

```diff
@@ -1,187 +1,144 @@
-# -*- coding: utf-8 -*-
-
+import pathlib
 
 import glob
 from xtgeo.xyz import Points
 from xtgeo.well import Well
-from xtgeo.common import XTGeoDialog
-
-xtg = XTGeoDialog()
-logger = xtg.basiclogger(__name__)
-
-if not xtg.testsetup():
-    raise SystemExit
 
-td = xtg.tmpdir
-TPATH = xtg.testpathobj
+WFILES1 = pathlib.Path("wells/reek/1/OP_1.w")
+WFILES2 = pathlib.Path("wells/reek/1/OP_[1-5]*.w")
 
-WFILES1 = TPATH / "wells/reek/1/OP_1.w"
-WFILES2 = TPATH / "wells/reek/1/OP_[1-5]*.w"
-WFILES3 = TPATH / "wells/reek/1/XP_*.w"
 
-
-def test_get_zone_tops_one_well():
+def test_get_zone_tops_one_well(testpath, tmp_path):
```

Comment:
I know about the tmp_path fixture in pytest; still I have avoided it as it is quite valuable to have all results in one folder (/TMP) instead of doing  a look and search for the output files somewhere in /tmp/. One reason for this is that often RMS is used as double check. Perhaps something we should discuss?

Reply:
I agree that sometimes it can be convenient to go over files that are created during a test - and it is indeed an additional step when having to locate the temporary directory. The `basebath` will be the same though, and within the randomly generated directory each test will receive their own directory as well.

Currently there are 1000 files generated in the TMP folder, at a total of 5.5 GB - with a majority of them generated with random name (so finding a file specific for a test could even be easier than it is today). The files are also never deleted, as we're not cleaning up our temp folder and thus we're aggregating the files. After four runs I'm at 11 GB and 1745 additional files in my source folder.

Do you think getting the files in specific folder per test name would outweigh the drawbacks of having the check for a random folder?

Reply:
Yes folders could help. Perhaps it is an idea to stick with the pytest fixture, and I see no major "big deal" that it is mixed in a period. 

By the way the TMP is cleaned when "python setup.py clean" is executed

Reply:
Glad to hear, I'll leave the commit as is then. And perhaps if it feels really awkward we could start out by seeing if we could fix it through pytest before reverting as well, there could very well be some tricks we could do to make our lives easier.

>and I see no major "big deal" that it is mixed in a period.

Yeah, I think doing this step by step is a good solution. There are a range of test review issues currently and we can do changes as we go through those instead of doing "it all at once".